code/modules/clothing/clothing.dm

```diff
@@ -521,6 +524,23 @@
 			else
 				to_chat(user, "<span class='notice'>[src] have already had [p_their()] toes cut open!</span>")
 		return
+
+	if(istype(I, /obj/item/kitchen/knife/combat))
+		var/obj/item/kitchen/knife/combat/W = I
```

Comment:
```suggestion
		var/obj/item/kitchen/knife/combat/W = I
```
Is this cast needed

Reply:
It seems not - copy/paste code logic moment :gatto: